core/src/main/java/io/delta/standalone/core/DeltaLogCore.java

```diff
@@ -0,0 +1,6 @@
+package io.delta.standalone.core;
+
+
+public interface DeltaLogCore {
```

Comment:
if this is an interface ... then are 'users' (connectors) supposed to instantiate `internal.DeltaLogCoreImpl`?

i.e. standalone provided a `DeltaLog.forTable` static API in its interface .. that could be used to instantiate the internal implementation